README.md

```diff
@@ -36,5 +37,121 @@ jupyter notebook
 
 **NOTE:** If the notebook depends on data files, you will need to
 download them explicitly if you don't use anaconda-project, by
-extracting the URLs defined in anaconda-project.yml and saving 
+extracting the URLs defined in anaconda-project.yml and saving
 the file(s) to this directory.
+
+# Uploading to AE
+In addition to running examples locally you can upload and share them
+using Anaconda Enterprise. If you've already installed anaconda-project,
+then for an example named "bears" just do:
+
+```
+cd bears
+anaconda-project archive bears.zip
+```
+
+Then in the AE interface select "Create", "Upload Project" and navigate
+to the zip file. Once your project has been created, you can deploy it.
+
+# Making a new project
+
+Once you have a notebook that you think it is ready to be its own project
+you can follow these steps to get it set up. For the examples I'll use
+an example project named "bears"
+
+### 1. Move the notebook into a new directory with same name
+
+```bash
+mkdir bears
+mv bears.ipynb ./bears
+cd bears
+```
+
+### 2. Get the minimum package dependencies
+
+I usually start by using nbrr (`conda install -c conda-forge nbrr `) which
```

Comment:
```suggestion
It can take a while to be sure you've captured every dependency, but I usually start by using nbrr (`conda install -c conda-forge nbrr `) which
```